Regression/prepare_file_travis.py

```diff
@@ -98,7 +99,17 @@ def select_tests(blocks, match_string_list, do_test):
             blocks = [ block for block in blocks if match_string in block ]
     return blocks
 
-if ci_regular_cartesian:
+if ci_regular_cartesian_2d:
+    test_blocks = select_tests(test_blocks, ['DIM=2'], True)
+    test_blocks = select_tests(test_blocks, ['USE_RZ=TRUE'], False)
+    test_blocks = select_tests(test_blocks, ['USE_PSATD=TRUE'], False)
+    test_blocks = select_tests(test_blocks, ['PYTHON_MAIN=TRUE'], False)
+    test_blocks = select_tests(test_blocks, ['PRECISION=FLOAT', 'USE_SINGLE_PRECISION_PARTICLES=TRUE'], False)
```

Comment:
Probably PEP8 line len:
```suggestion
    test_blocks = select_tests(test_blocks, ['PRECISION=FLOAT',
                               'USE_SINGLE_PRECISION_PARTICLES=TRUE'], False)
```